.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: Build
+
+on:
+  pull_request:
+    branches: ['**']
```

Comment:
tiny nitpick, feel free to take or leave - I don't think we need to specify `branches` here and under `push`.